<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>PlaylistFragment</class>
    <widget class="QFrame" name="PlaylistFragment">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1500</width>
                <height>741</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>1500</width>
                <height>741</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>1500</width>
                <height>741</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Frame</string>
        </property>
        <property name="styleSheet">
            <string notr="true">QFrame {
                background-color:#1e1d23;
                }
                QWidget {
                background-color:#1e1d23;
                }
                QMainWindow {
                background-color:#1e1d23;
                }
                QDialog {
                background-color:#1e1d23;
                }
                QColorDialog {
                background-color:#1e1d23;
                }
                QTextEdit {
                background-color:#1e1d23;
                color: #a9b7c6;
                }
                QPlainTextEdit {
                selection-background-color:#007b50;
                background-color:#1e1d23;
                border-style: solid;
                border-top-color: transparent;
                border-right-color: transparent;
                border-left-color: transparent;
                border-bottom-color: transparent;
                border-width: 1px;
                color: #a9b7c6;
                }
                QPushButton{
                border-style: solid;
                border-top-color: transparent;
                border-right-color: transparent;
                border-left-color: transparent;
                border-bottom-color: transparent;
                border-width: 1px;
                border-style: solid;
                color: #a9b7c6;
                padding: 2px;
                background-color: #1e1d23;
                }
                QPushButton::default{
                border-style: inset;
                border-top-color: transparent;
                border-right-color: transparent;
                border-left-color: transparent;
                border-bottom-color: #04b97f;
                border-width: 1px;
                color: #a9b7c6;
                padding: 2px;
                background-color: #1e1d23;
                }
                QToolButton {
                border-style: solid;
                border-top-color: transparent;
                border-right-color: transparent;
                border-left-color: transparent;
                border-bottom-color: #04b97f;
                border-bottom-width: 1px;
                border-style: solid;
                color: #a9b7c6;
                padding: 2px;
                background-color: #1e1d23;
                }
                QToolButton:hover{
                border-style: solid;
                border-top-color: transparent;
                border-right-color: transparent;
                border-left-color: transparent;
                border-bottom-color: #37efba;
                border-bottom-width: 2px;
                border-style: solid;
                color: #FFFFFF;
                padding-bottom: 1px;
                background-color: #1e1d23;
                }
                QPushButton:hover{
                border-style: solid;
                border-top-color: transparent;
                border-right-color: transparent;
                border-left-color: transparent;
                border-bottom-color: #37efba;
                border-bottom-width: 1px;
                border-style: solid;
                color: #FFFFFF;
                padding-bottom: 2px;
                background-color: #1e1d23;
                }
                QPushButton:pressed{
                border-style: solid;
                border-top-color: transparent;
                border-right-color: transparent;
                border-left-color: transparent;
                border-bottom-color: #37efba;
                border-bottom-width: 2px;
                border-style: solid;
                color: #37efba;
                padding-bottom: 1px;
                background-color: #1e1d23;
                }
                QPushButton:disabled{
                border-style: solid;
                border-top-color: transparent;
                border-right-color: transparent;
                border-left-color: transparent;
                border-bottom-color: #808086;
                border-bottom-width: 2px;
                border-style: solid;
                color: #808086;
                padding-bottom: 1px;
                background-color: #1e1d23;
                }
                QLineEdit {
                border-width: 1px; border-radius: 4px;
                border-color: rgb(58, 58, 58);
                border-style: inset;
                padding: 0 8px;
                color: #a9b7c6;
                background:#1e1d23;
                selection-background-color:#007b50;
                selection-color: #FFFFFF;
                }
                QLabel {
                color: #a9b7c6;
                }
                QLCDNumber {
                color: #37e6b4;
                }
                QProgressBar {
                text-align: center;
                color: rgb(240, 240, 240);
                border-width: 1px;
                border-radius: 10px;
                border-color: rgb(58, 58, 58);
                border-style: inset;
                background-color:#1e1d23;
                }
                QProgressBar::chunk {
                background-color: #04b97f;
                border-radius: 5px;
                }
                QMenuBar {
                background-color: #1e1d23;
                }
                QMenuBar::item {
                color: #a9b7c6;
                spacing: 3px;
                padding: 1px 4px;
                background: #1e1d23;
                }

                QMenuBar::item:selected {
                background:#1e1d23;
                color: #FFFFFF;
                }
                QMenu::item:selected {
                border-style: solid;
                border-top-color: transparent;
                border-right-color: transparent;
                border-left-color: #04b97f;
                border-bottom-color: transparent;
                border-left-width: 2px;
                color: #FFFFFF;
                padding-left:15px;
                padding-top:4px;
                padding-bottom:4px;
                padding-right:7px;
                background-color: #1e1d23;
                }
                QMenu::item {
                border-style: solid;
                border-top-color: transparent;
                border-right-color: transparent;
                border-left-color: transparent;
                border-bottom-color: transparent;
                border-bottom-width: 1px;
                border-style: solid;
                color: #a9b7c6;
                padding-left:17px;
                padding-top:4px;
                padding-bottom:4px;
                padding-right:7px;
                background-color: #1e1d23;
                }
                QMenu{
                background-color:#1e1d23;
                }
                QTabWidget {
                color:rgb(0,0,0);
                background-color:#1e1d23;
                }
                QTabWidget::pane {
                border-color: rgb(77,77,77);
                background-color:#1e1d23;
                border-style: solid;
                border-width: 1px;
                border-radius: 6px;
                }
                QTabBar::tab {
                border-style: solid;
                border-top-color: transparent;
                border-right-color: transparent;
                border-left-color: transparent;
                border-bottom-color: transparent;
                border-bottom-width: 1px;
                border-style: solid;
                color: #808086;
                padding: 3px;
                margin-left:3px;
                background-color: #1e1d23;
                }
                QTabBar::tab:selected, QTabBar::tab:last:selected, QTabBar::tab:hover {
                border-style: solid;
                border-top-color: transparent;
                border-right-color: transparent;
                border-left-color: transparent;
                border-bottom-color: #04b97f;
                border-bottom-width: 2px;
                border-style: solid;
                color: #FFFFFF;
                padding-left: 3px;
                padding-bottom: 2px;
                margin-left:3px;
                background-color: #1e1d23;
                }

                QCheckBox {
                color: #a9b7c6;
                padding: 2px;
                }
                QCheckBox:disabled {
                color: #808086;
                padding: 2px;
                }

                QCheckBox:hover {
                border-radius:4px;
                border-style:solid;
                padding-left: 1px;
                padding-right: 1px;
                padding-bottom: 1px;
                padding-top: 1px;
                border-width:1px;
                border-color: rgb(87, 97, 106);
                background-color:#1e1d23;
                }
                QCheckBox::indicator:checked {

                height: 10px;
                width: 10px;
                border-style:solid;
                border-width: 1px;
                border-color: #04b97f;
                color: #a9b7c6;
                background-color: #04b97f;
                }
                QCheckBox::indicator:unchecked {

                height: 10px;
                width: 10px;
                border-style:solid;
                border-width: 1px;
                border-color: #04b97f;
                color: #a9b7c6;
                background-color: transparent;
                }
                QRadioButton {
                color: #a9b7c6;
                background-color: #1e1d23;
                padding: 1px;
                }
                QRadioButton::indicator:checked {
                height: 10px;
                width: 10px;
                border-style:solid;
                border-radius:5px;
                border-width: 1px;
                border-color: #04b97f;
                color: #a9b7c6;
                background-color: #04b97f;
                }
                QRadioButton::indicator:!checked {
                height: 10px;
                width: 10px;
                border-style:solid;
                border-radius:5px;
                border-width: 1px;
                border-color: #04b97f;
                color: #a9b7c6;
                background-color: transparent;
                }
                QStatusBar {
                color:#027f7f;
                }
                QSpinBox {
                color: #a9b7c6;
                background-color: #1e1d23;
                }
                QDoubleSpinBox {
                color: #a9b7c6;
                background-color: #1e1d23;
                }
                QTimeEdit {
                color: #a9b7c6;
                background-color: #1e1d23;
                }
                QDateTimeEdit {
                color: #a9b7c6;
                background-color: #1e1d23;
                }
                QDateEdit {
                color: #a9b7c6;
                background-color: #1e1d23;
                }
                QComboBox {
                color: #a9b7c6;
                background: #1e1d23;
                }
                QComboBox:editable {
                background: #1e1d23;
                color: #a9b7c6;
                selection-background-color: #1e1d23;
                }
                QComboBox QAbstractItemView {
                color: #a9b7c6;
                background: #1e1d23;
                selection-color: #FFFFFF;
                selection-background-color: #1e1d23;
                }
                QComboBox:!editable:on, QComboBox::drop-down:editable:on {
                color: #a9b7c6;
                background: #1e1d23;
                }
                QFontComboBox {
                color: #a9b7c6;
                background-color: #1e1d23;
                }
                QToolBox {
                color: #a9b7c6;
                background-color: #1e1d23;
                }
                QToolBox::tab {
                color: #a9b7c6;
                background-color: #1e1d23;
                }
                QToolBox::tab:selected {
                color: #FFFFFF;
                background-color: #1e1d23;
                }
                QScrollArea {
                color: #FFFFFF;
                background-color: #1e1d23;
                }
                QSlider::groove:horizontal {
                height: 5px;
                background: #04b97f;
                }
                QSlider::groove:vertical {
                width: 5px;
                background: #04b97f;
                }
                QSlider::handle:horizontal {
                background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
                border: 1px solid #5c5c5c;
                width: 14px;
                margin: -5px 0;
                border-radius: 7px;
                }
                QSlider::handle:vertical {
                background: qlineargradient(x1:1, y1:1, x2:0, y2:0, stop:0 #b4b4b4, stop:1 #8f8f8f);
                border: 1px solid #5c5c5c;
                height: 14px;
                margin: 0 -5px;
                border-radius: 7px;
                }
                QSlider::add-page:horizontal {
                background: white;
                }
                QSlider::add-page:vertical {
                background: white;
                }
                QSlider::sub-page:horizontal {
                background: #04b97f;
                }
                QSlider::sub-page:vertical {
                background: #04b97f;
                }

                QPushButton{
                border-style: solid;
                border-color: #050a0e;
                border-width: 1px;
                border-radius: 5px;
                color: #d3dae3;
                padding: 2px;
                background-color: #100E19;
                }
                QPushButton::default{
                border-style: solid;
                border-color: #050a0e;
                border-width: 1px;
                border-radius: 5px;
                color: #FFFFFF;
                padding: 2px;
                background-color: #151a1e;
                }
                QPushButton:hover{
                border-style: solid;
                border-top-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #C0DB50, stop:0.4 #C0DB50,
                stop:0.5 #100E19, stop:1 #100E19);
                border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #100E19, stop:0.5
                #100E19, stop:0.6 #C0DB50, stop:1 #C0DB50);
                border-left-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #C0DB50, stop:0.3 #C0DB50,
                stop:0.7 #100E19, stop:1 #100E19);
                border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #C0DB50, stop:0.3
                #C0DB50, stop:0.7 #100E19, stop:1 #100E19);
                border-width: 2px;
                border-radius: 1px;
                color: #d3dae3;
                padding: 2px;
                }
                QPushButton:pressed{
                border-style: solid;
                border-top-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #d33af1, stop:0.4 #d33af1,
                stop:0.5 #100E19, stop:1 #100E19);
                border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #100E19, stop:0.5
                #100E19, stop:0.6 #d33af1, stop:1 #d33af1);
                border-left-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #d33af1, stop:0.3 #d33af1,
                stop:0.7 #100E19, stop:1 #100E19);
                border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #d33af1, stop:0.3
                #d33af1, stop:0.7 #100E19, stop:1 #100E19);
                border-width: 2px;
                border-radius: 1px;
                color: #d3dae3;
                padding: 2px;
                }
            </string>
        </property>
        <widget class="QLabel" name="l_Cover">
            <property name="geometry">
                <rect>
                    <x>420</x>
                    <y>120</y>
                    <width>200</width>
                    <height>200</height>
                </rect>
            </property>
            <property name="text">
                <string/>
            </property>
        </widget>
        <widget class="QPushButton" name="pb_user">
            <property name="geometry">
                <rect>
                    <x>670</x>
                    <y>270</y>
                    <width>391</width>
                    <height>61</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Segoe UI Black</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true"/>
            </property>
            <property name="text">
                <string/>
            </property>
        </widget>
        <widget class="QScrollArea" name="scrollArea">
            <property name="geometry">
                <rect>
                    <x>340</x>
                    <y>360</y>
                    <width>801</width>
                    <height>361</height>
                </rect>
            </property>
            <property name="widgetResizable">
                <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>799</width>
                        <height>359</height>
                    </rect>
                </property>
            </widget>
        </widget>
        <widget class="QPushButton" name="pb_changeCover">
            <property name="geometry">
                <rect>
                    <x>500</x>
                    <y>40</y>
                    <width>131</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="text">
                <string>Изменить обложку</string>
            </property>
        </widget>
        <widget class="QPushButton" name="pb_remove">
            <property name="geometry">
                <rect>
                    <x>1340</x>
                    <y>20</y>
                    <width>121</width>
                    <height>61</height>
                </rect>
            </property>
            <property name="text">
                <string>Удалить плейлист</string>
            </property>
        </widget>
        <widget class="QPushButton" name="pb_changeName">
            <property name="geometry">
                <rect>
                    <x>660</x>
                    <y>40</y>
                    <width>141</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="text">
                <string>Сохранить название</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="LE_name">
            <property name="geometry">
                <rect>
                    <x>670</x>
                    <y>180</y>
                    <width>371</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Segoe UI Black</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">border: none</string>
            </property>
        </widget>
        <widget class="QRadioButton" name="RB_Public">
            <property name="geometry">
                <rect>
                    <x>1350</x>
                    <y>100</y>
                    <width>141</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>11</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Публичный</string>
            </property>
        </widget>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>670</x>
                    <y>230</y>
                    <width>371</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Segoe UI</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="text">
                <string>Составитель:</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_2">
            <property name="geometry">
                <rect>
                    <x>670</x>
                    <y>120</y>
                    <width>371</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Segoe UI</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="text">
                <string>Плейлист</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
